azure-pipelines.yml

```diff
@@ -0,0 +1,83 @@
+trigger:
+  - azure*
```

Comment:
We probably want this to be `master`?

CI here is working because of PR triggers: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#pr-triggers  The default is to build all PRs